scripts/install-mender.sh

```diff
@@ -266,12 +281,48 @@ do_install_commercial() {
     echo "  Success!"
 }
 
+do_setup_mender() {
+    # Return if mender-client was not installed
+    if [[ ! "$SELECTED_COMPONENTS" == *"mender-client"* ]]; then
+        return
+    fi
+
+    # Return if no setup options were passed
+    if [ -z "$MENDER_SETUP_ARGS" ]; then
+        return
+    fi
+
+    echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
+    mender setup $MENDER_SETUP_ARGS
+    pidof systemd && systemctl restart mender-client
+    echo "  Success!"
```

Comment:
Suggestion:
echo "\tSuccess!" instead?

Reply:
Maybe. I was just following the current styling... I think I'll leave it as is.